specs/eid-hermes-theory-of-operation.md

```diff
@@ -29,11 +43,13 @@ specific slot against input data located in a specific program
 memory. The eBPF program may reference memory via offsets into the
 eBPF program memory. The eBPF program may populate parts of the eBPF
 program memory with both intermediate data *and* output data. Once the
-eBPF program has completed the device will inform the host (mechanism
-for this is TBD).
+eBPF program has completed the device will inform the host via an
+interrupt.
```

Comment:
Add: The host can also choose to poll the relevant BAR0 register to check for command completion.